bolts/data/datasets/audio/ecoacoustics_ds.py

```diff
@@ -0,0 +1,121 @@
+"""Ecoacostics dataset provided A. Eldridge et al.
+    Alice Eldridge, Paola Moscoso, Patrice Guyot, & Mika Peck. (2018).
+    Data for "Sounding out Ecoacoustic Metrics: Avian species richness
+    is predicted by acoustic indices in temperate but not tropical
+    habitats" (Final) [Data set].
+    Zenodo. https://doi.org/10.5281/zenodo.1255218
+"""
+from __future__ import annotations
+import os
+from os import listdir
+from os.path import isfile, join
+from pathlib import Path
+from typing import ClassVar, Optional, Union
+import zipfile
+
+import numpy as np
+import pandas as pd
+import torch
+from typing_extensions import Literal
+
+from bolts.data.datasets.audio.base import PBAudioDataset
+from bolts.data.datasets.utils import AudioTform, FileInfo
+
+__all__ = ["EcoacousticsDS"]
+SoundscapeAttr = Literal["habitat", "site"]
+
+
+class EcoacousticsDS(PBAudioDataset):
+    """Dataset for audio data collected in various geographic locations."""
+
+    LABELS_DIR: ClassVar[str] = "AvianID_AcousticIndices"
+    METADATA_FILENAME: ClassVar[str] = "metadata.csv"
+
+    _FILE_INFO: ClassVar[FileInfo] = FileInfo(name="EcoacousticsDS.zip", id="PLACEHOLDER")
+    _BASE_FOLDER: ClassVar[str] = "EcoacousticsDS"
+    _EC_LABELS_FILENAME: ClassVar[str] = "EC_AI.csv"
+    _UK_LABELS_FILENAME: ClassVar[str] = "UK_AI.csv"
+
+    def __init__(
+        self,
+        root: Union[str, Path],
+        *,
+        download: bool = True,
+        target_attr: SoundscapeAttr = "habitat",
+        transform: Optional[AudioTform] = None,
+    ) -> None:
+
+        self.root = Path(root)
+        self.download = download
+        self._base_dir = self.root / self._BASE_FOLDER
+        self.labels_dir = self._base_dir / self.LABELS_DIR
+        self._metadata_path = self._base_dir / "metadata.csv"
```

Comment:
maybe create a constant rather than keep using the `"metadata.csv"` string directly

Reply:
I actually already had a constant I just wasn't using it wow lol, good spot.